process-engine/bpmn-studio
# ✨ Add Overlays to Elements With Error

## Changes

1. Add Overlays to Elements With Error
2. Refactor LiveExecutionTrackerService
3. Fix Displaying of Notification in Case of Incorrectly Configured CallActivity
4. Add Tooltips to Overlays

PR: #1642

## How to test the changes

- Run a process with at least one element that creates an error
- **Notice that there will be a button (<img width="20" src="https://user-images.githubusercontent.com/20394992/62776590-474e5280-baab-11e9-8ef2-03c5ebe86908.png">) to navigate to the inspect-correlation view**
- Click on the button
- **Notice that the inspect-correlation view will be opened with the correct processInstance and the error element already selected.**

- Run a process with a call activity with an invalid target process
- **Notice that the two overlays will be displayed next to each other**
